electrum_nmc/electrum/gui/qt/forms/dnsdialog.ui

```diff
@@ -198,31 +207,6 @@
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
-         <!-- TODO: Reconcile this with Namecoin Core, which includes a "Hostname" field. -->
```

Comment:
I think these comments were removed automatically by the editor; please re-add them.